articles/quickstart/backend/rails/01-authorization.md

```diff
@@ -21,7 +21,7 @@ useCase: quickstart
 
 ### Install dependencies
 
-This tutorial performs Access Token validation using the  **jwt** Gem within a custom `JsonWebToken` class. A Concern called `Secured` is used to mark endpoints which require authentication through an incoming Access Token. 
+This tutorial performs Access Token validation using the  **[jwt](https://github.com/jwt/ruby-jwt)** Gem within a custom `Auth0Client` class. A Concern called `Secured` is used to authorize endpoints which require authentication through an incoming Access Token.
```

Comment:
```suggestion
This tutorial performs access token validation using the  **[jwt](https://github.com/jwt/ruby-jwt)** Gem within a custom `Auth0Client` class. A Concern called `Secured` is used to authorize endpoints which require authentication through an incoming access token.
```
The docs team's guidelines instruct to not case industry terms like 'access token'.